scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtConfig.scala

```diff
@@ -146,6 +146,7 @@ case class ScalafmtConfig(
     poorMansTrailingCommasInConfigStyle: Boolean = false,
     verticalMultilineAtDefinitionSite: Boolean = false,
     verticalMultilineAtDefinitionSiteArityThreshold: Int = 100,
+    verticalMultilineAtDefinitionSiteDanglingExclude: List[String] = List("class", "trait"),
```

Comment:
I propose we introduce a `VerticalMultiline` case class with the nested settings, similar to `Align` and `Newlines`. We can still keep the old `verticalMultilineAtDefinitionSite` `verticalMultilineAtDefinitionSiteArityThreshold` for backwards compat.

```
verticalMultiline {
  defnSite = true
  arityThreshold = 100
  ...
}
```

Reply:
Definitely, it was getting a bit messy 👍